vms/administrator/admin.py

```diff
@@ -1,6 +1,9 @@
-from django.contrib import admin
+#Django
 from administrator.models import Administrator
 
+# local Django
```

Comment:
I am not aware about how such comments help, can you please share with me @RiaPant, some writeup or something. :)

Reply:
@tapasweni-pathak Hey! I followed the official coding style PEP 8.

> Put imports in these groups: future, standard library, third-party libraries, other Django components, local Django component, try/excepts. Sort lines in each group alphabetically by the full module name. Place all import module statements before from module import objects in each section. Use absolute imports for other Django components and relative imports for local components.

Link: https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/

Reply:
Thanks for the information, merging it. 🚀